.github/workflows/package.yml

```diff
@@ -0,0 +1,31 @@
+name: Package Tarballs
+on:
+  # Run this workflow on branch master pushes and tag pushes that match v* (eg. `v1.2.3`)
+  push:
+    branch:
+      - 'master'
+    tags:
+      - 'v*'
+
+jobs:
+  package:
+    name: Package Tarballs
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Install Helm
+      uses: azure/setup-helm@v1
+      with:
+        version: v2.16.6
+
+    - name: Package tarballs
+      run: mkdir -p package; helm package --save=false --destination ./package ./conjur-oss
+
+    - name: Add package to artifacts
+      uses: actions/upload-artifact@v1
+      with:
+        path: ./package
+        name: package
```

Comment:
not a blocker but I feel like we should be creating a release instead of just uploading to artifacts but that can come later too

Reply:
@sgnn7 yeah, we can definitely go all the way to creating a release. the scope of this PR is limited to  having a pipeline. we should create an issue for anything beyond it